widgets/blog/styles/portfolio.less

```diff
@@ -50,7 +50,7 @@
 @featured_image_post_meta_font_weight: 600;
 @featured_image_post_meta_color: default;
 
-.sow-blog-layout-portfolio {
+.sow-blog-layout-portfolio .sow-blog-posts {
```

Comment:
@AlexGStapleton The Portfolio layout styling isn't working for me. This change doesn't look right.